<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RM_STR</class>
 <widget class="QMainWindow" name="RM_STR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>832</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RM_STR</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/img/browser.png</normaloff>:/img/browser.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <pointsize>1</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0,0,1">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="minimumSize">
                    <size>
                     <width>270</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>15</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Дата:</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLCDNumber" name="Date">
                   <property name="minimumSize">
                    <size>
                     <width>150</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::NoFrame</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <property name="smallDecimalPoint">
                    <bool>false</bool>
                   </property>
                   <property name="digitCount">
                    <number>8</number>
                   </property>
                   <property name="mode">
                    <enum>QLCDNumber::Dec</enum>
                   </property>
                   <property name="segmentStyle">
                    <enum>QLCDNumber::Flat</enum>
                   </property>
                   <property name="intValue" stdset="0">
                    <number>120518</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="minimumSize">
                    <size>
                     <width>270</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>15</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string> Технологический номер:</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineTechNumber">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>150</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>16</pointsize>
                    </font>
                   </property>
                   <property name="mouseTracking">
                    <bool>true</bool>
                   </property>
                   <property name="text">
                    <string>0000000000</string>
                   </property>
                   <property name="frame">
                    <bool>true</bool>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="clearButtonEnabled">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="label_SUB">
                   <property name="font">
                    <font>
                     <pointsize>15</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>SUB-20:</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="SwitchButton" name="b_sub" native="true">
                   <property name="statusTip">
                    <string>Вкл/выкл sub-20</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_RLC">
                   <property name="font">
                    <font>
                     <pointsize>15</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>RLC E7-20:</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="SwitchButton" name="b_rlc" native="true">
                   <property name="statusTip">
                    <string>Вкл/выкл RLC E7-20</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="Line" name="line_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <widget class="wManualControl" name="wManControl"/>
          <widget class="wAutoControl" name="pAutoControl"/>
          <widget class="wProc" name="pProtocol"/>
          <widget class="QWidget" name="pDatabase">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="wDatabase" name="wDB" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1162</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="manageDB"/>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="manualCheck"/>
   <addaction name="separator"/>
   <addaction name="autoCheck"/>
   <addaction name="separator"/>
   <addaction name="protocolCheck"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <action name="manualCheck">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/settings-1.png</normaloff>:/img/settings-1.png</iconset>
   </property>
   <property name="text">
    <string>Ручная проверка</string>
   </property>
   <property name="statusTip">
    <string>Ручная проверка</string>
   </property>
   <property name="whatsThis">
    <string>Ручная проверка</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="autoCheck">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/settings.png</normaloff>:/img/settings.png</iconset>
   </property>
   <property name="text">
    <string>Автоматическая проверка</string>
   </property>
   <property name="statusTip">
    <string>Автоматическая проверка</string>
   </property>
   <property name="whatsThis">
    <string>Автоматическая проверка</string>
   </property>
  </action>
  <action name="protocolCheck">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/invoice.png</normaloff>:/img/invoice.png</iconset>
   </property>
   <property name="text">
    <string>Протокол проверки</string>
   </property>
   <property name="statusTip">
    <string>Протокол проверки</string>
   </property>
   <property name="whatsThis">
    <string>Протокол проверки</string>
   </property>
  </action>
  <action name="exit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/exit.png</normaloff>:/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="statusTip">
    <string>Закрытие программы</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/wrench.png</normaloff>:/img/wrench.png</iconset>
   </property>
   <property name="text">
    <string>Настройки и оборудование</string>
   </property>
   <property name="statusTip">
    <string>Настройка RLC E7-20 и SUB-20</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="manageDB">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/server.png</normaloff>:/img/server.png</iconset>
   </property>
   <property name="text">
    <string>Настройка и редактирование БД</string>
   </property>
   <property name="statusTip">
    <string>Редактирование БД</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/about.png</normaloff>:/img/about.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="statusTip">
    <string>О программе</string>
   </property>
   <property name="whatsThis">
    <string>О программе</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>switchbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>wDatabase</class>
   <extends>QWidget</extends>
   <header>wdatabase.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>wManualControl</class>
   <extends>QWidget</extends>
   <header>wmanualcontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>wAutoControl</class>
   <extends>QWidget</extends>
   <header>wautocontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>wProc</class>
   <extends>QWidget</extends>
   <header>wproc.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
